js.es5.md

```diff
@@ -1,6 +1,6 @@
-# K&L JavaScript Style Guide
+# Education ES5 JavaScript Style Guide
```

Comment:
Yeah, so I would say those are features of the language rather than anything to do with style.